templates/event/sponsor-request.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'global/base.html' %}
+{% load staticfiles %}
+
+{% block title %}Request for GitHub sponsoring{% endblock %}
+{% block description %}Fill the form available on this page to apply for a GitHub grant.{% endblock %}
+
+{% block content %}
```

Comment:
Can you fix the indentation of that block? It seems to be indented by one level too much.